### PR TITLE
Added Jobs page and two community resources (SoybeanBase and SCNBase) under Community

### PR DESCRIPTION
**Pull request by weihuang12**

### CODE REVIEW

**Review by StevenCannon-USDA — COMMENTED** [review]
It looks OK to me, but I think it might be better to link to http://localhost:4000/about/ (which has the contact form) than to link to the team page.

**Comment by weihuang12** [comment]
Thanks for the suggestion @StevenCannon-USDA , I have made the link to https://localhost:4000/about/.

File: community/jobs/index.html
Comment (StevenCannon-USDA) [inline community/jobs/index.html:1]: Sorry - one more minor change to make:
`If you have a soybean-related job to announce, please contact <a href= "/about/" target="_blank">SoyBase</a> to get it posted.`

Comment (weihuang12) [reply community/jobs/index.html:1]: Thanks @StevenCannon-USDA . Changes have made.

**Review by StevenCannon-USDA — APPROVED** [review]

**Comment by maxglycine** [comment]
Looks good to me. I see it has been merged, but it looks ok to me

**Pull request merged**